docs/settings_ref.rst

```diff
@@ -180,6 +180,24 @@ Allows you to set a leeway of the JWT token. See the official
 `PyJWT <https://pyjwt.readthedocs.io/en/stable/usage.html>`__ docs for more information.
 
 
+FAILED_RESPONSE_FUNCTION
+--------------------------------
+* **Default**: ``None``
```

Comment:
This isn't actually the default value. It should either be `default_failed_response` or the lambda.

Reply:
Changed to `lambda`